docs/src/support/README.md

```diff
@@ -1,3 +1,70 @@
 # Getting Help #
 
-Page for information around where to look for help and get support.
+## I'm having trouble logging into DataLabs ##
+
+In order to login to DataLabs you'll need to create an account using the sign up button
+from the home page. Once you've created an account, you'll need to verify your e-mail
+before being able to sign in.
+
+## I have signed in but can't see anything, what do I do now ##
+
+Initially you will only be able to see projects which you own or have been given
+access to. If you are collaborating with someone else on a project, you can ask them to
+add you to the relevant project. Otherwise, get in touch with us on Slack about
```

Comment:
How generic does the documentation need to be? Do we need to handle that this should soon be on CaaS where we are likely not going to be in control of the instance? I.e. do we need to change it so that is is more along the lines of 

> Otherwise, contact your DataLabs instance administrator to request a project.